web/component2/src/demo_components/prop_validation.dart

```diff
@@ -19,25 +19,25 @@ part 'prop_validation.over_react.g.dart';
 UiFactory<PropTypesTestProps> PropTypesTest = castUiFactory(_$PropTypesTest); // ignore: undefined_identifier
 
 mixin PropTypesTestProps on UiProps {
-  int count;
-  List twoObjects;
-  int defaultCount;
-  bool content;
+  int? count;
+  List? twoObjects;
+  int? defaultCount;
+  bool? content;
 
-  bool hideHeader;
+  bool? hideHeader;
   dynamic header;
 
-  List initiallyExpandedKeys;
+  List? initiallyExpandedKeys;
   dynamic targetKey;
 
-  bool hideLabel;
-  String label;
+  bool? hideLabel;
+  String? label;
 }
 
 class PropTypesTestComponent extends UiComponent2<PropTypesTestProps> {
   @override
   render() =>
-      Fragment()(Dom.code()('props.twoObjects'), ' currently has ${props.twoObjects.length} objects passed to it.');
+      Fragment()(Dom.code()('props.twoObjects'), ' currently has ${props.twoObjects!.length} objects passed to it.');
```

Comment:
Sorry, I think we discussed it briefly in a previous comment, but I think we should update examples and web in a follow-up PR since it's a decent chunk of work. Made a ticket for that: FED-1931